peg/php/PType.hx

```diff
@@ -4,10 +4,13 @@ enum PType {
 	TInt;
 	TFloat;
 	TString;
+	TArrayOfString;
```

Comment:
`TArray` should be a generic type. `TArray(type:PType)`

Reply:
done in 09f1f4c - I've implemented a recursion to handle this.